tests/acceptance/features/apiOcisSpecific/apiShareWebdavOperations-propfind.feature

```diff
@@ -6,4 +6,4 @@ Feature: PROPFIND
   Scenario: PROPFIND to "/remote.php/dav/files"
```

Comment:
If you check the line above you should find the ` after fixing all issues delete this Scenario and use the one from oC10 core`, meaning that AFAICT this complete feature file can be removed, as it is covered by the core tests.

- [ ] remove `tests/acceptance/features/apiOcisSpecific/apiShareWebdavOperations-propfind.feature`